sources/packages/web/src/services/ScholasticStandingService.ts

```diff
@@ -55,6 +56,19 @@ export class ScholasticStandingService {
     };
   }
 
+  /**
+   * Get Scholastic Standing Summary details.
+   * @param studentId student id to retrieve the scholastic standing.
```

Comment:
studentId should not be optional and the comment is not mentioning the `options`.